src/js/chartAuthoring.js

```diff
@@ -40,6 +42,27 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
                         "onTemplateInjected.escalate": "{chartAuthoring}.events.onChartAuthoringInterfaceReady.fire"
                     },
                     components: {
+                        chartTitle: {
+                            type: "fluid.inlineEdit",
+                            container: "{chartAuthoring}.dom.chartTitle",
+                            createOnEvent: "{chartAuthoring}.events.onChartAuthoringInterfaceReady",
+                            options: {
+                                strings: {
+                                    defaultViewText: "Enter Chart Title"
```

Comment:
This definition should be surfaced as top ```strings``` options of ```floe.chartAuthoring``` so it's more configurable. Pass the top option down using distribute options or this:
```
strings: {
    defaultViewText: "{floe.chartAuthoring}.options.strings.defaultTitleText"
}
```

The same comment applies to line 61.